README.md

```diff
@@ -19,7 +19,8 @@ into [CDI](http://www.cdi-spec.org/) beans,or [JAX-RS](https://github.com/eclips
 
 ## Branches
 
-* main - 3.x development stream. Uses Vert.x 4.x and Microprofile 4.x
+* main - 4.x development stream. Uses Vert.x 4.x, Microprofile 5.x and Jakarta 9
```

Comment:
Isn't it 10? 
Should we mention Flow?

Reply:
I thought so at first. But 10 is in for MP 6.

Reply:
For flow I'll rebase the PR #1896 and merge it. We can mention it there.